### DISCUSSION TITLE
[resolved] Definiowanie tabeli rozmiarów

### DISCUSSION THREAD

**Discussion by tomasz-grzesik** [allegro/allegro-api#4413] [Suggestions - resolved]
:bulb: **Sugestia**

Z dokumentacji wynika, że jest tylko możliwość pobrania tabeli rozmiarów.
Brakuje możliwości utworzenia nowej tabeli rozmiarów.

**Comment by hahyes** [comment]
Podbijam, chciałbym móc korzystać masowo z tej możliwości, jednak API na to nie pozwala. A szkoda.
